### PR TITLE
Added missing test-libfuzzer.sh scripts

### PR DESCRIPTION
**Pull request by adrianherrera**

### CODE REVIEW

File: guetzli-2017-3-30/test-libfuzzer.sh [hunk 1]
```diff
@@ -5,6 +5,7 @@
 set -x
 rm -rf $CORPUS
 
+mkdir $CORPUS
 cp -r $SCRIPT_DIR/seeds $CORPUS
```
Comment (morehouse) [inline guetzli-2017-3-30/test-libfuzzer.sh]: I think we should remove the `cp` and pass both `$CORPUS` and the seeds directory to libFuzzer below.

File: llvm-libcxxabi-2017-01-27/test-libfuzzer.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Copyright 2016 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+set -x
+. $(dirname $0)/../common.sh
+
+rm -rf $CORPUS
```
Comment (morehouse) [inline llvm-libcxxabi-2017-01-27/test-libfuzzer.sh]: We probably also need to remove any fuzzing logs in these scripts.

Comment (adrianherrera) [reply llvm-libcxxabi-2017-01-27/test-libfuzzer.sh]: Done!

**Comment by adrianherrera** [comment]
Hey @morehouse, all updated!

**Review by morehouse — APPROVED** [review]

**Pull request merged**